puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-installer-develop-release.groovy

```diff
@@ -0,0 +1,83 @@
+def commit_hash = ''
+def sourcefile = ''
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+
+    environment {
+        ruby_version = "2.4"
+    }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                dir('foreman-installer') {
+                    git(url: 'https://github.com/theforeman/foreman-installer.git', branch: 'develop')
+                    script { commit_hash = sh(script: "git rev-parse HEAD | tee commit", returnStdout: true).trim() }
```

Comment:
I'm wondering if there's a built in variable for the git hash.

Reply:
I didn't look too hard yesterday before doing it myself, but I'd be surprised if there wasn't one.

Reply:
I think there is but only if you check out the project at the top level. Given you can check out multiple git repositories, Jenkins doesn't have a clean way to know which project to populate the git hash with. This might be worth a library function:

https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman_packaging_release.groovy#L65
https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tfmROR52.groovy#L92

Reply:
updated